spec/openapi.yaml

```diff
@@ -9330,8 +9330,7 @@ paths:
               type: string
             type: array
         - description: >-
-            Optional parameter to filter to the specified list of translation
-            resources.
+            Optional parameter of unique agency identifier.
```

Comment:
Description says that this is agency related parameter but parameter name is `userUids`